dev_readme.md

```diff
@@ -2,6 +2,8 @@
 
 #### 1. Download and install [Docker](https://www.docker.com/products/docker-desktop) for your operating system.
 
+You might find the chapter on [reproducible computational environments](https://the-turing-way.netlify.app/reproducible-research/renv.html) and spectifically the section on [containers](https://the-turing-way.netlify.app/reproducible-research/renv/renv-containers.html) in _The Turing Way_ useful!
```

Comment:
This is demonstrating a review! 🌟

Reply:
Can this be hyperlinked to a subsection within the `Extra Tips` section? 

I'd like to keep the instructions minimalistic.

Reply:
Can this be hyperlinked to a subsection within the `Extra Tips` section? 

I'd like to keep the instructions minimalistic.